llumnix/server_info.py

```diff
@@ -11,6 +11,57 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+class RequestStatistics:
```

Comment:
consider using a namedtuple

Reply:
The value of namedtuple cannot be changed while we should change the value of timestamp in each step,  and we should use @property function to calculate the latency according to timestamp, so RequestStatistics should be class.